test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -2549,10 +2451,10 @@ func TestActivatePods(t *testing.T) {
 	}
 }
 
-func initTestSchedulerForFrameworkTest(t *testing.T, testCtx *testutils.TestContext, nodeCount int, opts ...scheduler.Option) *testutils.TestContext {
+func initTestSchedulerForFrameworkTest(t *testing.T, testCtx *testutils.TestContext, nodeCount int, opts ...scheduler.Option) (*testutils.TestContext, testutils.ShutdownFunc) {
```

Comment:
Can you add a comment explaining under which circumstances the shutdown function should get called and when?

Some tests seem to ignore it (e.g. `TestActivatePods`).

Reply:
How was this resolved?